provider/vsphere/internal/vsphereclient/createvm_test.go

```diff
@@ -384,6 +385,35 @@ func (s *clientSuite) TestCreateVirtualMachineRootDiskSize(c *gc.C) {
 	})
 }
 
+func (s *clientSuite) TestVerifyMAC(c *gc.C) {
+	var testData = []struct {
+		Mac    string
+		Result bool
+	}{
+		{"foo:bar:baz", false},
+		{"00:22:55:11:34:11", false},
+		{"00:50:56:123:11:11", false},
+		{"00:50:56:40:12:23", false},
+		{"00:50:56:3f:ff:ff", true},
+		{"00:50:56:12:34:56", true},
+		{"00:50:56:2A:eB:Cd", true},
+	}
+	for i, test := range testData {
+		c.Logf("test #%d: MAC=%s expected %s", i, test.Mac, test.Result)
+		c.Check(VerifyMAC(test.Mac), gc.Equals, test.Result)
+	}
+}
+
+// GenerateMAC is random so we verify a few times if the MACs returned are
+// are correct.
+func (s *clientSuite) TestGenerateMAC(c *gc.C) {
```

Comment:
These seems a little odd to run. We should feel confident generatemac should always generate a valid mac address. Our larger concern should be around verifymac, and we could explore edge cases to confirm.

I don't think we need this.

Reply:
I just want to be sure that we won't have a situation in which someone in the future decides that 4M MACs is not enough and we should make more bits random.